src/components/common/LeftMainPanel.tsx

```diff
@@ -0,0 +1,16 @@
+import { ReactElement } from "react";
+import HamburguerMenu from "./HamburgerMenu";
+
+const LeftMainPanel = (props: { children: ReactElement }) => {
```

Comment:
I think this works and `ts` doesn't scream:

```suggestion
const LeftMainPanel = ({ children}: ReactElement) => {
```

Reply:
Same here with the brackets in children